system/jlib/jlzw.cpp

```diff
@@ -1830,6 +1832,8 @@ typedef enum { ICFcreate, ICFread, ICFappend } ICFmode;
 #define COMPRESSEDFILEFLAG (I64C(0xc0528ce99f10da55))
 #define COMPRESSEDFILEBLOCKSIZE (0x10000)
 #define FASTCOMPRESSEDFILEFLAG (I64C(0xc1518de99f10da55))
+#define LZ4COMPRESSEDFILEFLAG (I64C(0xc00000099f10da55))
+#define DENCOMPRESSEDFILEFLAG (I64C(0xc00000198f10db47))
```

Comment:
I tried to figure out how the previous was derived and I could not, so I selected some unique numbers at random.  I wondered if there was any pattern or mask or such but could not find any.